src/ert/_c_wrappers/enkf/lark_parser.py

```diff
@@ -45,6 +46,25 @@ def __str__(self):
         return self.value
 
 
+def assert_is_token(
```

Comment:
I think this should be renamed to `_assert_is_token` to make it clear that this is private to the file.